tools/klee/main.cpp

```diff
@@ -622,8 +622,12 @@ std::string KleeHandler::getRunTimeLibraryPath(const char *argv0) {
     KLEE_DEBUG_WITH_TYPE("klee_runtime", llvm::dbgs() <<
                          "Using build directory KLEE library runtime :");
     libDir = KLEE_DIR;
+#ifdef KLEE_CMAKE_BUILD
```

Comment:
I've decided there shouldn't be any source code changes because the CMake build is going to need to be fixed so we can support building the runtime in different build modes. So please remove the `KLEE_CMAKE_BUILD` define and instead set `RUNTIME_CONFIGURATION` appropriately (it can be a dummy value for now).